core/templates/components/state-editor/state-editor.directive.html

```diff
@@ -14,8 +14,8 @@ <h3 class="oppia-exp-content-card-header">Content</h3>
   </div>
   <div class="oppia-mobile-collapsible-card-content">
     <div class="oppia-editor-card-body oppia-editor-concept-container" ng-if="conceptCardIsShown">
-      <img ng-if="!windowIsNarrow" ng-src="<[oppiaBlackImgUrl]>" alt="" class="oppia-editor-card-avatar">
-      <div id="tutorialStateContent" class="oppia-editor-card-section state-content-editor-parent-container">
+      <img ng-if="!windowIsNarrow" ng-src="<[oppiaBlackImgUrl]>" alt="" class="oppia-editor-card-avatar" ng-class="{'oppia-editor-card-avatar-sticky': isSticky}">
+      <div id="tutorialStateContent" class="oppia-editor-card-section" ng-class="{'state-content-editor-parent-container': !isSticky}">
```

Comment:
Will this affect the state editor too?

Reply:
Just saw. I missed responding to this. The code you're referring to is for reducing the height of the question header in the review modal after it becomes sticky. It is only going to affect the question review modal.

Reply:
Yes, I had verified that for not only the exploration editor, but also for question suggestion editor, and question-list in the topic and skill editor. The code affected only the question suggestion review modal.